src/PythonProjectBootstrapper/EntryPoint.py

```diff
@@ -234,10 +235,20 @@ def _ExecuteOutputDir(
             accept_hooks=True,
         )
 
-    CopyToOutputDir(src_dir=tmp_dir, dest_dir=output_dir)
+    modifications = CopyToOutputDir(src_dir=tmp_dir, dest_dir=output_dir)
+
+    prompt_text_path = PathEx.EnsureFile(output_dir / "prompt_text.yml")
 
     if not skip_prompts:
-        DisplayPrompt(output_dir=output_dir)
+        with open(prompt_text_path, "r") as prompt_file:
+            prompts = yaml.load(prompt_file, Loader=yaml.Loader)
+
+        prompt_text_path.unlink()

```

Comment:
This line is repeated in both conditions of the if clause. It can be written once with an `ExitStack`.